python/sglang/srt/models/deepseek_v2.py

```diff
@@ -591,7 +592,7 @@ def __init__(
 
         self.top_k = config.num_experts_per_tok
 
-        if get_moe_a2a_backend().is_deepep():
+        if not get_moe_a2a_backend().is_none():
```

Comment:
I prefer `is_deepep` or `is_mooncake` here as we will support other a2a backends.